xbmc/guilib/GUITexture.cpp

```diff
@@ -216,27 +234,37 @@ void CGUITexture::Render()
   if (m_info.border.x1)
   {
     if (m_info.border.y1)
-      Render(m_vertex.x1, m_vertex.y1, m_vertex.x1 + m_info.border.x1, m_vertex.y1 + m_info.border.y1, 0, 0, u1, v1, u3, v3);
-    Render(m_vertex.x1, m_vertex.y1 + m_info.border.y1, m_vertex.x1 + m_info.border.x1, m_vertex.y2 - m_info.border.y2, 0, v1, u1, v2, u3, v3);
+      Render(m_vertex.x1, m_vertex.y1, m_vertex.x1 + m_info.border.x1,
+             m_vertex.y1 + m_info.border.y1, 0, 0, u1, v1, u3, v3, 0, 0, du1, dv1, du3, dv3);
+    Render(m_vertex.x1, m_vertex.y1 + m_info.border.y1, m_vertex.x1 + m_info.border.x1,
```

Comment:
extra whitespace may increase readability from the if statement

Reply:
Like this?